src/working-mode

```diff
@@ -0,0 +1,242 @@
+   in any way on the implementers, however.)
+  </ul>
+
+  <p>Additionally, the following are strongly recommended:
+
+  <ul>
+   <li>The support from implementers should be of the form "we would like to implement this soon"
+   and not just "this seems like a reasonable idea".
+
+   <li>There should be no strong implementer objections to the new feature.
+
+   <li>There should already be a prototype implementation or one being worked on side-by-side with
+   the change to the standard.
+
+   <li>New features should be something that cannot be done today or are such a common trend in
+   libraries that standardizing them helps lots of people. Alternatively, the "new" feature could
+   actually be a legacy feature that was not yet standardized, which is now being standardized. (A
+   legacy "addition" is important to allow for more meaningful competition and interoperability.)
+  </ul>
+
+  <h4>Removals</h4>
+
+  <p>Any change that represents a removal needs to meet these additional criteria:
+
+  <ul>
+   <li><p>The feature being removed must either be not widely implemented, or must in the process of
+   being removed from implementations.
+
+   <li>
+    <p>A test ensuring the feature is not supported must be added and existing tests for the feature
+    should be adjusted or removed as appropriate.
+
+    <p class="example"><a href="https://github.com/w3c/web-platform-tests/pull/5001">w3c/web-platform-tests#5001</a>
+    added new tests that appcache was not supported in shared workers, per the removal in
+    <a href="https://github.com/whatwg/html/pull/2384">whatwg/html#2384</a>.
+
+    <p class="example">Adjusting existing tests can be difficult. If necessary, an issue can be
+    filed instead to track updating those tests. This was done in
+    <a href="https://github.com/w3c/web-platform-tests/issues/5053">w3c/web-platform-tests#5053</a>
+    which accompanied the removal in
+    <a href="https://github.com/whatwg/html/pull/2402">whatwg/html#2402</a>.
+  </ul>
+
+  <h3>New Proposals</h3>
+
+  <p>As described above, the criteria for inclusion in a WHATWG standard is rather strict. In the
+  initial stages of feature development for the web platform, such widespread implementer support is
+  often not available, or the shape of a feature is not yet clear enough for implementers to feel
+  comfortable pledging their support.
+
+  <p>In such scenarios we anticipate features will be "incubated" outside of WHATWG standards. This
+  could be in a variety of venues, such as:
+
+  <ul>
+   <li><p>Another standards organization
+   <li><p>A personal document or GitHub repository
+   <li><p>A pull request to a WHATWG Standard, with the understanding it will not be merged until
+   the criteria above are met
+   <li>
+    <p>A document hosted by the WHATWG, but not under the spec.whatwg.org subdomain and not titled
+    "Living Standard"
+
+    <p class="example"><a href="https://wiki.whatwg.org/wiki/CanvasColorSpace">CanvasColorSpace</a>.
```

Comment:
I'd expand this to a full sentence, such as "The [CanvasColorSpace] proposal is being incubated on [the WHATWG wiki]."